src/libraries/System.Private.CoreLib/src/System/Reflection/NullabilityInfoContext.cs

```diff
@@ -385,12 +386,14 @@ private NullabilityInfo GetNullabilityInfo(MemberInfo memberInfo, Type type, ILi
 
                 for (int i = 0, offset = 0; i < genericArguments.Length; i++)
                 {
-                    if (!genericArguments[i].IsValueType)
+                    Type t = Nullable.GetUnderlyingType(genericArguments[i]) ?? genericArguments[i];
+
+                    if (!t.IsValueType || t.Name.Contains(ValueTuple))
```

Comment:
Any type that includes the text "ValueTuple" anywhere in its name gets special-cased?

Reply:
For the context: for generics type parameters `NullableAttribute` byte array is missing the byte for a value type, but not for ValueTuple, i expect it is happening only for ValueTuple type, i will change to `StartsWith` instead of `Contains`, not sure how else i could improve this condition.

Reply:
StartsWith isn't appropriate either.  It'll still match other types you're not trying to match, and if there's an issue with ValueTuple, there's likely an issue with other types as well.

I suggest better investigating what the actual problem is and why it's affecting ValueTuple, and then using that increased knowledge to come up with a proper fix.

Reply:
Ah right, it should be a generic type, not only `ValueTuple`, thank you!

Reply:
And added tests for generic struct